examples/pxScene2d/src/pxScene.cpp

```diff
@@ -15,7 +15,11 @@
 #include "pxUtil.h"
 
 #ifdef RUNINMAIN
+#ifdef ENABLE_DEBUG_MODE
+extern rtNode* script;
```

Comment:
Let's either make script always a rtNode* or make it always an object (no matter if ENABLE_DEBUG_MODE is enabled or not)